ckanext/data_qld/plugin.py

```diff
@@ -2,15 +2,18 @@
 
 import logging
 
-from ckan import plugins
-
-from . import actions, auth_functions as auth, blueprints, click_cli, \
-    constants, converters, datarequest_auth_functions, helpers, validation
 import ckantoolkit as tk
 
+from ckan import model
```

Comment:
These two lines could be combined: `from ckan import model, plugins`

Reply:
Fixed